students/Wayne/Assignments/Lesson2/PrinterGridExercise.py

```diff
@@ -0,0 +1,91 @@
+#print grid with single variable
+
+# declaring grid variables
+
+plus = '+'
+
+endplus = '+'
+
+minus = ' - '
```

Comment:
This'll make the squares wider, with two spaces between each dash instead of one.